mcs/class/System.Net.Http/HttpClientHandler.wasm.cs

```diff
@@ -0,0 +1,148 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Net.Security;
+using System.Security.Authentication;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.Net.Http
+{
+	public partial class HttpClientHandler : HttpMessageHandler
+	{
+		HttpMessageHandler wasmHandler;
+
+		public HttpClientHandler () : this (HttpClient.CreateDefaultHandler ()) { }
+
+		HttpClientHandler (HttpMessageHandler wasmHandler)
+		{
+			this.wasmHandler = wasmHandler;
+		}
+
+		protected override void Dispose (bool disposing)
+		{
+			if (disposing) {
+				if (wasmHandler != null) {
+					wasmHandler.Dispose ();
+					wasmHandler = null;
+				}
+			}
+			base.Dispose (disposing);
+		}
+
+		const string EXCEPTION_MESSAGE = "System.Net.Http.HttpClientHandler is not supported on the current platform.";
+
+		public virtual bool SupportsAutomaticDecompression => throw new PlatformNotSupportedException (EXCEPTION_MESSAGE);
```

Comment:
bool properties should all be `false` instead of throw

Reply:
Done.